### ISSUE TITLE
vmui: add tab for viewing raw data

### ISSUE THREAD

**Issue by hagen1778** [VictoriaMetrics/VictoriaMetrics#7024]
### Is your feature request related to a problem? Please describe

Users are regularly confused with the way how [Query API](https://docs.victoriametrics.com/keyconcepts/#query-data) works in VictoriaMetrics or Prometheus. It is mistakenly believed that Query API returns data as it is, when in reality data samples and their timestamps can be modified by the API.
In result, users may have problems with understanding why they're seeing one results or another. In [our docs](https://docs.victoriametrics.com/troubleshooting/#unexpected-query-results) we recommend to simplify the query, or event export its raw representation in order understand the results.

### Describe the solution you'd like

It would be great to introduce a new View tab in vmui for displaying Raw Data. 

This idea was initially presented by Bryan Boreham on Promcon 2024 via just executing `metric[$__range]` query in Grafana's explore mode using [instant query API](https://docs.victoriametrics.com/keyconcepts/#instant-query) and `time series` formatting of the output. The result visually showed how the data looks like in a raw format - [see example](https://play-grafana.victoriametrics.com/explore?schemaVersion=1&panes=%7B%22vnh%22:%7B%22datasource%22:%22PE8D8DB4BEE4E4B22%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22go_memory_classes_heap_unused_bytes%5B$__range%5D%5Cn%22,%22range%22:false,%22instant%22:true,%22datasource%22:%7B%22type%22:%22prometheus%22,%22uid%22:%22PE8D8DB4BEE4E4B22%22%7D,%22editorMode%22:%22code%22,%22legendFormat%22:%22__auto%22,%22exemplar%22:false,%22interval%22:%22%22%7D%5D,%22range%22:%7B%22from%22:%22now-15m%22,%22to%22:%22now%22%7D%7D%7D&orgId=1).

I see it as a new View tab in vmui. This tab should have a field for entering time series selector (note, only selector not query expression) and a graph for displaying the result. The time range `$__range` should be automatically added to the selector based on selected time range. The tab should make requests to /api/v1/export endpoint. 
The tab should have an option to disable deduplication feature by adding `reduce_mem_usage=1` GET param to requests (see https://docs.victoriametrics.com/single-server-victoriametrics/#how-to-export-data-in-json-line-format).

**Comment by hagen1778** [comment]
cc @valyala

**Comment by Haleygo** [comment]
Why not instruct users to send an instant query `timeseries_selector[$__range]` using existed `Table` option like [this](https://play.victoriametrics.com/select/accounting/1/6a716b0f-38bc-4856-90ce-448fd713e3fe/prometheus/graph/#/?g0.range_input=30m&g0.end_input=2024-10-23T07%3A34%3A54&g0.relative_time=last_30_minutes&g0.tab=1&g0.expr=up%5B1m%5D)? It also has `Trace query` to enable.

**Comment by hagen1778** [comment]
> Why not instruct users to send an instant query timeseries_selector[$__range] using existed Table option like [this](https://play.victoriametrics.com/select/accounting/1/6a716b0f-38bc-4856-90ce-448fd713e3fe/prometheus/graph/#/?g0.range_input=30m&g0.end_input=2024-10-23T07%3A34%3A54&g0.relative_time=last_30_minutes&g0.tab=1&g0.expr=up%5B1m%5D)? It also has Trace query to enable.

Because a human-being can't spot gaps between timestamps in table view :)

**Comment by hagen1778** [comment]
@Loori-R I'd really appreciate this for debugging purposes.

**Comment by bartbutenaers** [comment]
> It is mistakenly believed that Query API returns data as it is

I can confirm this as a VictoriaMetrics newbie.  I started devoloping a few days ago an integration for Node-RED (= open source IOT tool).  Have been looking for many hours why my code 'seemed' to be ingesting so many datapoints.  I was getting insane at the end 😉

Yesterday I finally found that both the query API and the user interface are working completely different from what my brain had expected.  And yes now I see that it written is in the documentation, so it is completely my own mistake.

But it would really help to show it more visual, that VictoriaMetrics works like this.

Thanks a lot!!!
Bart
